microsoft-edge/webview2/samples/webview2apissample.md

```diff
@@ -19,188 +19,324 @@ The **WebView2APISample** app demonstrates how to use the WebView2 control and W
+<!-- ---------- -->
+###### Visual Studio 2017
+
+If you want to open this sample by using Visual Studio 2017: 
+
+1. Open the solution in Visual Studio 2017.
 
-1. If you previously cloned the repo, pull the latest commits to your local copy of the repo.
+1. Change the project's Platform Toolset in **Project Properties > Configuration properties > General > Platform Toolset**.
+
+1. You might also need to install a recent Windows SDK on your machine.
 
 
 <!-- ====================================================================== -->
-## Step 3 - Open the solution in Visual Studio
+## Step 2: Clone the WebView2Samples repo
 
-1. On your local drive, open the `.sln` file in Visual Studio:
+1. Press **Windows + E**.
 
-   *  `<your-repos-directory>/WebView2Samples/SampleApps/WebView2Samples.sln`
+   A File Explorer window opens.
 
-   or:
+1. Navigate to where you cloned or plan to clone the **WebView2Samples** repo.
 
-   *  `<your-repos-directory>/WebView2Samples-main/SampleApps/WebView2Samples.sln`
+   Example path:
 
-The **WebView2APISample** sample and project is the main Win32 sample.
+   `C:\Users\localAccount\GitHub\WebView2Samples`
 
-Unlike some other samples, there's not a dedicated `.sln` file in the sample repo directory that contains this sample's Readme.  Instead, the `.sln` file for this sample (including other sample projects as well) is in the parent directory.
+1. If not done already, clone the `WebView2Samples` repo to your local drive.  In a separate window or tab, see [Clone the WebView2Samples repo](../how-to/machine-setup.md#clone-the-webview2samples-repo) in _Set up your Dev environment for WebView2_.  Follow the steps in that section, and then return to this page and continue below.
 
-![All projects in the solution in Solution Explorer](./webview2apissample-images/all-projects-in-solution-explorer.png)
+1. If not opened already, open a git bash command prompt: Press the **Windows** key, and then type **git bash**.
+
+
+   **Fetch and pull commits**
+
+1. Switch to the cloned repo directory:
+
+   ```
+   cd C:/Users/localAccount/GitHub/WebView2Samples`
+   ```
+
+   Substitute the path where you cloned the **WebView2Samples** repo to.
+
+1. Fetch the latest repo information:
+
+   ```
+   git fetch
+   ```
+
+1. Pull the latest commits to your local copy of the repo:
+
+   ```
+   git pull
+   ```
+
+
+   **Create a working branch and switch to it**
+
+1. Create a working branch:
+
+   ```
+   git branch test
+   ```
+
+1. Switch to the working branch:
+
+   ```
+   git checkout test
+   ```
+
+   Returns: `Switched to branch 'test'`
+
+You are now free to modify the code in your working branch, without altering the code that's in the "main" branch of the repo.  Later you might want to switch back to the "main" branch, or create a different branch based off the "main" branch.
```

Comment:
```suggestion
You are now free to modify the code in your working branch, without altering the code that's in the "main" branch of the repo.  Later you might want to switch back to the "main" branch, for example to pull new commits, or create a different branch based off the "main" branch.
```

Reply:
not the best example? bc:
if you're in "test" branch and issue "git pull" cmd, it pulls commits for all branches?

Reply:
added the phrase